tests/benchmarks/current_consumption/system_off/boards/nrf54l15dk_nrf54l15_cpuapp_gpio_wakeup_ret_mem.overlay

```diff
@@ -0,0 +1,58 @@
+/ {
```

Comment:
I propose to not mix features into single overlay, rather use nrf54l15dk_nrf54l15_cpuapp_gpio_wakeup.overlay
and add overlay for retention (only that feature in the overlay).
Then at sample.yaml, use both of those overlays.
This also prevents duplication of content of nrf54l15dk_nrf54l15_cpuapp_gpio_wakeup.overlay.

Reply:
done